manifests/kustomize/base/cache-deployer/cache-deployer-deployment.yaml

```diff
@@ -4,10 +4,12 @@ metadata:
   name: cache-deployer-deployment
   labels:
     app: cache-deployer
+    application-crd-id: kubeflow-pipelines
```

Comment:
I see there are a couple of instances where you have added the labels manually, may I ask why? why not put this in the kustomization.yaml via: 

```
labels:
- includeSelectors: true
  pairs:
    application-crd-id: kubeflow-pipelines
```

like you have done elsewhere?

Reply:
hey @HumairAK, initially i was updating the files manually and not using the prescribed method used from `kustomize`

Reply:
let me move all the ones i've added manually to `kustomization.yaml`